.buildkite/x-pack/pipeline.xpack.winlogbeat.yml

```
@@ -295,3 +298,21 @@ steps:
         notify:
           - github_commit_status:
               context: "x-pack/winlogbeat: Packaging Linux"
+
+      - label: ":ubuntu: x-pack/winlogbeat: Packaging Linux arm64"
```

```suggestion
      - label: ":ubuntu: x-pack/winlogbeat: Packaging Windows arm64"
```

Same as above, we are producing a windows artifact so let's name the step after that regardless of what the host machine OS is.